src/types/constraint.ts

```diff
@@ -3,7 +3,7 @@ import type { TimeRange } from './timeline';
 export type Constraint = {
   created_at: string;
   definition: string;
-  description: string;
+  description?: string;
```

Comment:
What is the reason for making this optional? Since the [description](https://github.com/NASA-AMMOS/aerie/blob/develop/merlin-server/sql/merlin/tables/constraint.sql#L5) has a default and should never be null, it should always be present.

Reply:
I don't have the ability to double check right now, but I saw some code that inserts a constraint with potentially no `description` in the object. I only saw it because of strict=true.

Reply:
Ah in that case we need to use the `ConstraintInsertInput` type instead I think since it's an insert

Reply:
Thanks @camargo! It looks like it's the effect function that isn't strict enough then. I've updated it to not be optional